contrib/storage-googlesheets/src/main/java/org/apache/drill/exec/store/googlesheets/GoogleSheetsStoragePlugin.java

```diff
@@ -61,6 +62,7 @@ public class GoogleSheetsStoragePlugin extends AbstractStoragePlugin {
   private final OAuthTokenProvider tokenProvider;
   private DataStore<StoredCredential> dataStore;
   private Sheets service;
+  private Drive driveService;
```

Comment:
Do these need to be closed at some point?

Reply:
@jnturton I don't think they need to be closed.  These objects are really more like containers for the various bits that go into calling the Google APIs.  They don't actually open any input streams, at least I don't think they do.  You can see that when we actually pull data from Google, there is some convoluted call which ends in `execute`.  That makes the actual HTTP call.

Reply:
Let me verify that before we merge this.

Reply:
Confirmed.  The objects in question don't have a `close` method.